geometric_data/ocean/transect/Nares_Strait/transect.geojson

```diff
@@ -1,42 +1,27 @@
 {
-    "type": "FeatureCollection", 
+    "type": "FeatureCollection",
     "features": [
         {
-            "type": "Feature", 
+            "type": "Feature",
             "properties": {
-                "name": "Nares Strait", 
-                "tags": "Nares_Strait;Critical_Passage", 
-                "object": "transect", 
-                "component": "ocean", 
-                "author": "Xylar Asay-Davis", 
-                "depth": "250.0"
-            }, 
+                "name": "Nares Strait",
+                "tags": "standard_transport_sections",
+                "object": "transect",
+                "component": "ocean",
+                "author": "Milena Veneziani",
```

Comment:
@mark-petersen, I believe you altered this feature so you should add yourself as an author.

```suggestion
                "author": "Milena Veneziani, Mark Petersen",
```

Reply:
@milenaveneziani, are you okay with the movement of the transect a bit further north?

Reply:
@xylar: I have lengthened the Nares transect inland by quite a bit, avoiding the small channel and without shifting it north. I have chosen the location of the transects to where I know there is observations to compare to. Before the final merge, it will be a good idea to re-create masks for a high res and a low mesh, just to make sure.
One other thing I wanted to say is whether we want to add some transects in the SO. I didn't do a thorough search for observations, but  the search I did do didn't produce many results.

Reply:
Ah, got it.  So @mark-petersen's version is no longer there and he no longer needs to be a coauthor.  I think yours is the cleaner solution.

Regarding the SO, I'm not aware of good transects.  The best we're likely to get are a few moorings (e.g. at the Filchner Trough and various Amundsen Sea troughs) but I don't have a lot of time to figure out what the corresponding transects would be or to ask for the data right now.  Let's plan to add them later.